src/components/SubscriptionSwitchesList/SubscriptionSwitchesList.js

```diff
@@ -55,8 +58,14 @@ const SubscriptionSwitchesList = ({
     return (
       <MyAccountError
         icon={happyData}
-        title={t('Subscription switch in progress!')}
-        subtitle={t('Please try again in a few moments.')}
+        title={t(
+          'subscription-switches-list.switch-in-progress',
+          'Subscription switch in progress!'
+        )}
+        subtitle={t(
+          'subscription-switches-list.try-again',
+          'Please try again in a few moments.'
+        )}
         margin="0 auto"
```

Comment:
in this file, where you have a list of switches - I would suggest to add unique key for button text, as publishers may want to change 'update' / 'downgrade' button to simply 'Choose'